apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -259,6 +280,20 @@ export default {
     noResharePermsMessage () {
       const translated = this.$gettext('You don\'t have permission to share this %{type}.')
       return this.$gettextInterpolate(translated, { type: this.highlightedFile.type }, false)
+    },
+
+    currentUsersPermissions () {
+      if (this.incomingShares.length > 0) {
+        let permissions = 1
```

Comment:
reminds me that we still don't have constants for perms like https://github.com/owncloud/core/blob/master/core/js/js.js#L51

Reply:
We do. I just forgot to use it here - fixed